src/rust/src/x509/ocsp_req.rs

```diff
@@ -95,7 +95,10 @@ impl OCSPRequest {
 
     #[getter]
     fn serial_number<'p>(&self, py: pyo3::Python<'p>) -> Result<&'p pyo3::PyAny, PyAsn1Error> {
-        Ok(big_asn1_uint_to_py(py, self.cert_id()?.serial_number)?)
+        Ok(big_byte_slice_to_py_int(
+            py,
+            self.cert_id()?.serial_number.as_bytes(),
+        )?)
```

Comment:
This is uncovered because of silliness. assign `self.cert_id()?.serial_number.as_bytes()` to a local var